tests/Oryx.BuildImage.Tests/Node/NodeJsSampleAppsOtherTests.cs

```diff
@@ -42,6 +42,25 @@ public static TheoryData<string> ImageNameData
             }
         }
 
+        [Fact, Trait("category", "latest")]
```

Comment:
Basically our strategy is to split tests by its build image type, does it resolve the category overlapping issue you run into last time? I can see that most of changes happened in build image tests, do we have a plan to do same thing for the integration tests in the night builds, cause I see we also modify the nightly.yml file for it?

Reply:
It would be great to share what would be the estimated time after the optimization? Thanks!

Reply:
Thanks @qianz2! Yes I've got a workaround more information on this comment: https://github.com/microsoft/Oryx/pull/1438/files#r906511480

This modification is for all pipeline that build BuildImages (Validation, Nightly, CI). The plan is to do similar work for Build and Test Runtime Images too. My plan was to check that separately.

We reduced the time for BuildImages by more than half 😃 I put more details in the description.